<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDRMainClass</class>
 <widget class="QMainWindow" name="SDRMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDRMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1081</width>
      <height>961</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="SwitchesShow" name="switchesShowWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1081</width>
       <height>941</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>361</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QPushButton" name="pushButton_importconfig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Import Configuration (from switches.json)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exportconfig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Configuration (to switches.json)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_changLine">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>changeLine</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_changLinex1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_changLiney1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>y1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_changLiney2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>y2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_changLinestat">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>stat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_changLinex2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>x2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_changLinenum">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>num</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_testButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TestButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeView" name="treeView_ESP32s">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>430</y>
      <width>361</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SwitchesShow</class>
   <extends>QWidget</extends>
   <header>SwitchesShow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SDRMain.qrc"/>
 </resources>
 <connections/>
</ui>
